litgpt/data/base.py

```diff
@@ -79,11 +79,9 @@ def __getitem__(self, idx: int) -> Dict[str, Tensor]:
         if self.transform is not None:
             example = self.transform(example)
         prompt = self.prompt_style.apply(prompt=example["instruction"], **example)
-        prompt_and_response = prompt + example["output"]
         encoded_prompt = self.tokenizer.encode(prompt, max_length=self.max_seq_length)
-        encoded_prompt_and_response = self.tokenizer.encode(
-            prompt_and_response, eos=True, max_length=self.max_seq_length
-        )
+        encoded_response = self.tokenizer.encode(example["output"], eos=True, max_length=self.max_seq_length)
```

Comment:
By default, encode only adds a bos if the config specifies it should do so. Therefore, after these changes you need to set 
```suggestion
        encoded_response = self.tokenizer.encode(example["output"], eos=True, bos=False, max_length=self.max_seq_length)
```

here and remove this logic:

```
encoded_response[1:]
```